nixos/modules/services/networking/kea.nix

```diff
@@ -168,7 +207,17 @@ in
             type = listOf str;
             default = [];
             description = ''
-              List of additonal arguments to pass to the daemon.
+              Kea DHCP-DDNS configuration as a path, see <link xlink:href="https://kea.readthedocs.io/en/kea-${package.version}/arm/ddns.html"/>.
```

Comment:
Repurposing `extraArgs` for that use case looks like a copy&paste mistake.

Reply:
Shoot, that was indeed a copy&paste mistake